components/GymPlan/WorkoutSet.js

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+
+export default function WorkoutSet({ setName }) {
+  return (
+    <>
+      <StyledSection>
+        <h3> {setName} </h3>
+        <form>
+          <input type="number" name="set" label="set" placeholder="2" />
```

Comment:
Hey Kirsten. 
There are no Labels for all of your inputs. 
Please make sure to have Labels for a11y reasons.

Reply:
Hi Julia, wenn das label um die input Felder gepackt wird. <label> <input/> </label> wäre das ein Weg, wenn nicht unbedingt eine Textbeschreibung angezeigt werden soll?